app/controllers/home_controller.rb

```diff
@@ -1,6 +1,13 @@
 class HomeController < ApplicationController
   def index
-    # dummy data to build out the front end
+
+    projects = Project.all.order(start_date: :desc)
+    unique_donors = projects.uniq{|x| x.donors}.map{ |e| e.donors }.sort
```

Comment:
An easier and quicker way of doing this would be: `projects.pluck(:donors).uniq`
`pluck` extracts the required information from each object and then returns an array, so, from that array, you can do `.uniq`.

Same thing applies for the filters below

Reply:
And then you can apply `.sort` as well of course

Reply:
Sweet thanks for that! I'll fix the code and push it again.